mdgen/src/main/java/uk/gov/ida/mdgen/MetadataGenerator.java

```diff
@@ -76,23 +80,34 @@
     @CommandLine.Parameters(index = "1", description = "YAML definition file")
     private File yamlFile;
 
-    @CommandLine.Parameters(index = "2", description = "Public X509 cert for saml signing")
-    private File samlSigningCertFile;
-
-    @CommandLine.Parameters(index = "3", description = "Public X509 cert for metadata signing")
+    @CommandLine.Parameters(index = "2", description = "Public X509 cert for metadata signing")
     private File metadataSigningCertFile;
 
+    @CommandLine.Option(names = "--hsm-metadata-signing-label", description = "HSM Metadata key label", required = true)
```

Comment:
```suggestion
    @CommandLine.Option(names = "--hsm-metadata-signing-key-label", description = "HSM metadata signing key label", required = true)
```

Reply:
@Wynndow is going to alter these as part of his merge - changing the cli arguments will break the go code.

Reply:
👍